lib/tic_tac_toe_api_web/controllers/page_controller.ex

```diff
@@ -1,22 +1,35 @@
 defmodule TicTacToeApiWeb.PageController do
   use TicTacToeApiWeb, :controller
 
+  @seed Application.get_env(:tic_tac_toe_api, :seed)
+
   def index(conn, _params) do
     board = %Board{}
     render(conn, "index.json", board: board)
   end
 
   def status(conn, %{"spots" => spots, "next_player" => next_player, "current_player" => current_player}) do
-    result = build_result(spots, next_player, current_player)
+    result = board_with_players(spots, next_player, current_player)
+             |> Status.result(next_player, current_player)
     render(conn, "status.json", result: result)
   end
 
-  defp build_result(spots, next_player, current_player) do
-    player1 = %Player{symbol: next_player}
-    player2 = %Player{symbol: current_player}
-    spots_list = Jason.decode!(spots)
-                 |> ReactAdapter.swap_marks_with_players(player1, player2)
-    Status.result(spots_list, next_player, current_player)
+  defp board_with_players(spots, player1, player2) do
+    player1 = %Player{symbol: player1}
+    player2 = %Player{symbol: player2}
+    Jason.decode!(spots)
+    |> ReactAdapter.swap_marks_with_players(player1, player2)
   end
 
+  def computer(conn, %{"spots" => spots, "computer" => computer, "human" => human}) do
+    new_board = board_with_players(spots, computer, human)
+
+    if @seed == "" do
```

Comment:
Oh, I see what's going on. I missed this line during my original review and originally thought that you were still passing the empty string see into the randomizer.

It sounds like for the tests you want to have deterministic behavior, which is good for testing, but bad for the random computer. What are some ways you could either inject the computer type or leverage the configuration to determine what type of computer to use?

Reply:
We just update the code to leverage the configuration; does this look right?

Reply:
I don't think my comments contradict what Eric has said here, but feel free to ping me if you have any questions. I agree with all of the comments that Eric has made in this PR